linker/shared/src/main/scala/org/scalajs/linker/frontend/BaseLinker.scala

```diff
@@ -171,16 +167,12 @@ final class BaseLinker(config: CommonPhaseConfig) {
           assert(m.body.isDefined,
               s"The abstract method ${classDef.name.name}.${m.encodedName} " +
               "is reachable.")
-          val linked = linkedMethod(m)
-          if (m.name.isInstanceOf[Ident])
-            methods += linked
-          else
-            exportedMembers += linked
+          methods += linkedMethod(m)
         }
 
-      case m: PropertyDef =>
+      case m: JSMethodPropDef =>
         if (analyzerInfo.isAnySubclassInstantiated)
-          exportedMembers += linkedProperty(m)
+          exportedMembers += new Versioned(m, None)
```

Comment:
Hum, shouldn't we forward the version of `JSMethodDef`s here?

Reply:
Oops, indeed. Fixed.